docs/samples/deployments/spark/presidio_anonymize_blobs.py

```diff
@@ -0,0 +1,156 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Anonymize PII Entities in text files
+# MAGIC
+# MAGIC <br>Using Presidio, anonymize PII content of files in an Azure Storage account.
+# MAGIC
+# MAGIC <br>The following code sample will:
+# MAGIC <ol>
+# MAGIC <li>Import the content of text files located in an Azure Storage blob folder</li> # noqa D501
+# MAGIC <li>Anonymize the content using Presidio</li>
+# MAGIC <li>Write the anonymized content back to the Azure Storage blob account</li>
+# MAGIC </ol>
+
+# COMMAND ----------
+
+from azure.storage.blob import BlobServiceClient
+from presidio_analyzer import AnalyzerEngine
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import AnonymizerConfig
+from pyspark.sql.types import StringType
+from pyspark.sql.functions import input_file_name, regexp_replace
+from pyspark.sql.functions import col, pandas_udf
+import pandas as pd
+
+dbutils.widgets.text("storage_account_name", "", "Blob Storage Account Name")
+dbutils.widgets.text("storage_container_name", "", "Blob Container Name")
+dbutils.widgets.text("storage_account_access_key", "", "Storage Account Access Key")
+dbutils.widgets.text("storage_input_folder", "input", "Input Folder Name")
+dbutils.widgets.text("storage_output_folder", "output", "Output Folder Name")
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC # Import the text files from Azure Blob storage
+
+# COMMAND ----------
+
+storage_account_name = dbutils.widgets.get("storage_account_name")
+storage_container_name = dbutils.widgets.get("storage_container_name")
+storage_account_access_key = dbutils.widgets.get("storage_account_access_key")
+
+# mount the container
+spark.conf.set(
```

Comment:
This config only needs to be set when accessing blob storage directly i.e. without mounting. When mounting the `dbutils.fs.mount()` method is enough. In order to avoid any access issues I also recommend to run an adhoc `spark.conf.remove("fs.azure.account.key." + storage_account_name + ".blob.core.windows.net")` in your own Databricks environment.

Reply:
using that to init the blobservice client for uploading. if we can't write single output files, or choose to output the sample as batched string files, we can remove this. 
@omri374 wdyt?